### DISCUSSION TITLE
Wide spaces between classes/decimals (thousands, millions) in score numbers

### DISCUSSION THREAD

**Discussion by Loreos7** [ppy/osu#14517] [Q&A] [answered]
How it looks like in English version:
--
![ScreenShot_20210826131452](https://user-images.githubusercontent.com/86934170/130946890-1a2b3637-b087-4659-8b12-937de1b3d6af.png)
How it does in Russian, Swedish, Serbian, Slovenian, Romanian, Polish, Hungarian, French, Finnish, Czech, Belarusian, Bulgarian:
--
![ScreenShot_20210826131532](https://user-images.githubusercontent.com/86934170/130946869-b9c7d0fe-f4b4-4240-afc3-4270b7e0ecc5.png)

Is it intentional? 
--

**Comment by frenzibyte** [comment]
That's intentional, correct. Not all languages use comma for thousands separator, some use dots, some use spaces. Read https://en.wikipedia.org/wiki/Decimal_separator for more information.

**Reply by Loreos7** [reply]
But in my opinion they are too large. Need to shorten that spaces by a half. Looks much more readable. For example:
![ScreenShot_20210826131532](https://user-images.githubusercontent.com/86934170/130959291-019ac976-0e72-4af8-a4f8-4ead5c8ce8a2.jpg)

**Reply by frenzibyte** [reply]
You weren't clear about the issue you were reporting at first, almost as if you were asking whether it's intended that some locales use spaces rather than commas.

**Comment by Loreos7** [comment]
> You weren't clear about the issue you were reporting at first, almost as if you were asking whether it's intended that some locales use spaces rather than commas.

You didn't notice. The title says: "LARGE spaces". And they are REALLY large so i asked is it intentional. I know about decimal separators (spaces, ",", and dots). That amount of space makes numbers really hard to read. Just compare this
![ScreenShot_20210826184650](https://user-images.githubusercontent.com/86934170/130994336-379dc265-6672-4a59-b31f-121883caa74c.png)
![ScreenShot_20210826184633](https://user-images.githubusercontent.com/86934170/130994341-c5ca4693-faad-4401-adb5-61ec00b94764.png)

My feelings tell me that spaces should be shorten by a half.

**Reply by Loreos7** [reply]
![image](https://user-images.githubusercontent.com/86934170/130994817-1146dc9a-2266-4ed4-82a8-d1dc5d522965.png)

From wikipedia how it should be

**Reply by frenzibyte** [reply]
The word "large" in the title alone doesn't really mean that the spaces are larger than expected, but either way I understand what you're reporting now, that looks to be a problem.

**Reply by Loreos7** [reply]
Wide, sorry for bad eng

**Comment by frenzibyte** [comment, marked as answer]
Resolved in https://github.com/ppy/osu-framework/pull/4734.

**Reply by Loreos7** [reply]
Love u.
